sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/android/PackageName.kt

```diff
@@ -1,16 +1,27 @@
 package app.cash.sqldelight.gradle.android
 
 import com.android.build.gradle.BaseExtension
-import com.android.ide.common.symbols.getPackageNameFromManifest
+import com.android.build.gradle.internal.manifest.parseManifest
```

Comment:
This sounds risky? What if it's not available on previous versions of the AGP or if it gets removed later on?

Reply:
i think every AGP bump I end up switching to whatever new API exists to parse the manifest 😅 One day I will find one that is stable

Reply:
i should check which versions of AGP its on though :+1: